sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -306,7 +306,7 @@ private[hive] class SparkExecuteStatementOperation(
         parentSession.getSessionState.getConf.setClassLoader(executionHiveClassLoader)
       }
 
-      val substitutorStatement = new VariableSubstitution(sqlContext.conf).substitute(statement)
+      val substitutorStatement = new VariableSubstitution().substitute(statement)
```

Comment:
is `sqlContext.conf` the active conf here?

Reply:
I wrapped this statement in `SQLConf.withExistingConf`  to be on the safe side.